addons/mail/static/src/components/chatter/chatter.js

```diff
@@ -60,6 +60,10 @@ class Chatter extends Component {
         return this.env.models['mail.chatter'].get(this.props.chatterLocalId);
     }
 
+    mounted() {
+        this.chatter.thread.markNeedactionMessagesAsRead();
+    }
```

Comment:
Prefer fix in models instead, `_onChangeThreadViews` is already doing similar stuff.

Also make sure RPC is not done if there's no needaction.

Reply:
This is clearing ALL the notifications !
My guess is that `_onChangeThreadViews` is computed even when you don't open the thread

Reply:
Yes, all computes are always called at create, that's why there is a condition on threadView > 0

Reply:
I think you need to exclude mailbox from this. But please return to the compute version, not mounted.

Actually you need to exclude all non-origin thread, as per done in https://github.com/odoo/odoo/pull/62981